.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should add `.idea` to the `.gitignore` file so that your editor doesn't check in files that don't belong in the repo.

Reply:
Thanks for catching this. Resolved in https://github.com/openforcefield/openforcefield/pull/119/commits/dea744de97d776e6b6ed02618efe023d6ab3decb